[Resubmit] Don't crash on retrieveDesyncReport

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #98470
* #97702
* #97692
* #97544
* #97462

Per title

Differential Revision: [D44736409](https://our.internmc.facebook.com/intern/diff/D44736409/)

rohan-varma committed:
[Resubmit] Don't crash on retrieveDesyncReport

Per title

Differential Revision: [D44736409](https://our.internmc.facebook.com/intern/diff/D44736409/)

[ghstack-poisoned]

rohan-varma committed:
Update on "[Resubmit] Don't crash on retrieveDesyncReport"


Per title

Differential Revision: [D44736409](https://our.internmc.facebook.com/intern/diff/D44736409/)

[ghstack-poisoned]

XilunWu reviewed (approved):
give it a pass since it is a resubmit.

rohan-varma committed:
Update on "[Resubmit] Don't crash on retrieveDesyncReport"


Per title

Differential Revision: [D44736409](https://our.internmc.facebook.com/intern/diff/D44736409/)

[ghstack-poisoned]

rohan-varma committed:
Update on "[Resubmit] Don't crash on retrieveDesyncReport"


Per title

Differential Revision: [D44736409](https://our.internmc.facebook.com/intern/diff/D44736409/)

[ghstack-poisoned]

rohan-varma commented:
@pytorchbot merge -f "CI unrelated"
